SQL/MySQL.sublime-syntax

```diff
@@ -123,20 +123,27 @@ contexts:
     - meta_prepend: true
     - match: \b(?i:create\s+or\s+replace)\b
       scope: keyword.other.ddl.sql
-      push: [ddl-create-target, create-condition, ddl-target]
```

Comment:
I think there's a mild case for `-`-lists, if only to see at a glance how deep the stack becomes. I'm fine with this change, but I'd also be okay with the original.